llama_hub/llama_packs/arize_phoenix_query_engine/README.md

```diff
@@ -0,0 +1,105 @@
+<center>
+    <p style="text-align:center">
+        <img alt="phoenix logo" src="https://storage.googleapis.com/arize-assets/phoenix/assets/phoenix-logo-light.svg" width="200"/>
+        <br>
+        <a href="https://docs.arize.com/phoenix/">Docs</a>
+        |
+        <a href="https://github.com/Arize-ai/phoenix">GitHub</a>
+        |
+        <a href="https://join.slack.com/t/arize-ai/shared_invite/zt-1px8dcmlf-fmThhDFD_V_48oU7ALan4Q">Community</a>
+    </p>
+</center>
+<h1 align="center">Arize-Phoenix LlamaPack</h1>
+
+This LlamaPack instruments your LlamaIndex application for LLM application tracing with [Phoenix](https://github.com/Arize-ai/phoenix), an open-source LLM observability product from [Arize AI](https://arize.com/).
+
+## Install and Import Dependencies
+
+Install dependencies.
+
+
+```python
+!pip install "arize-phoenix[llama-index]" llama-hub html2text
+```
+
+Import libraries.
+
+
```

Comment:
largely not needed text

Reply:
addressed in [9911a6b](https://github.com/logan-markewich/llama-hub/pull/2/commits/9911a6ba9c8b1a261a07a2e3a86cd30aa00d259e)